### PR TITLE
Archie/red status widget

### PR DESCRIPTION
**Pull request by archiepeterjarvis**
# Refactor Spindle & Vacuum buttons

[Jira Ticket](<link_to_jira_ticket>)
- [x] Ready for review

## Checklist
- [x] I have completed a self review
- [ ] I have updated the [spreadsheet](https://docs.google.com/spreadsheets/d/1277nPOKtBFOk_kCo870_zA460-qlw_kGnYlLPwtnadE/edit#gid=0)
- [x] I have updated the jira ticket
- [x] I have added the relevant labels to this PR
- [ ] I have updated documentation (if applicable)
- [ ] I have run the unit tests suite and they pass
- [ ] I have updated the [change log](https://docs.google.com/document/d/1t3s6lQuUcug1Fj8E_i2gBJNvXN0-funwBArG2hsia6k/edit)

## Description
This branch was built off of archie/refactor_blinking_widget, as I was previously using assets that had the Red Cross on top of them, but this caused the images to not be in the centre of the blinking widget, so I'm just adding one on top of the image instead.

### Visual Test
- [ ] Not applicable
- [ ] Own computer
- [x] Console 7"
- [x] Console 10"

### Function Test
- [ ] Not applicable
- [ ] Own computer
- [x] Console 7"
- [x] Console 10"

### Unit Tests
- [ ] Not applicable
- [ ] Completed

## Screenshots (if applicable)

### CODE REVIEW

File: src/asmcnc/comms/router_machine.py [hunk 1]
```diff
@@ -1185,35 +1185,36 @@ def correct_rpm(self, requested_rpm, spindle_voltage = None, revert = False):
 
         return rpm_to_set
 
-    def turn_on_spindle(self, rpm=None):
+    def power_on_spindle(self):
```
Comment (Masc999) [inline src/asmcnc/comms/router_machine.py:1188]: I find it hard to see the difference from "power_on" to "turn_on". This is only used in rare occasions anyway so we could just call turn_on(rpm=0) there. So it is clear that we start with no RPM. No need for a 1-line function.

File: src/asmcnc/comms/router_machine.py [hunk 2]
```diff
@@ -1860,19 +1861,9 @@ def jog_relative(self, axis, dist, speed):
...
 
-    def spindle_on(self):
-        self.s.write_command('M3 S12000')
-
-    def spindle_off(self):
-        self.s.write_command('M5')
-
     def cooldown_zUp_and_spindle_on(self):
         self.s.write_command('AE')
-        if self.spindle_voltage == 230:
```
Comment (Masc999) [inline src/asmcnc/comms/router_machine.py:1871]: wow. this code was pretty useless... :)

File: src/asmcnc/production/z_head_qc_jig/z_head_qc_2.py [hunk 3]
```diff
@@ -275,16 +275,16 @@ def read_info_again(dt):
...
             else:
                 fail_report.append("Spindle brush time after reset was " + str(self.m.s.spindle_brush_run_time_seconds) + ". Should be 0")
-                self.m.s.write_command('M5')
+                self.m.turn_off_spindle()
                 self.test_rpm(fail_report)
 
         fail_report = []
         self.brush_reset_test_count += 1
-        self.m.s.write_command('M3 S0')
+        self.m.power_on_spindle()  # Turn on spindle to read info (at 0 rpm)
```
Comment (Masc999) [inline src/asmcnc/production/z_head_qc_jig/z_head_qc_2.py:287]: like above. would turn this into m.turn_on_spindle(rpm=0)

File: src/asmcnc/skavaUI/img/extraction_off.png
Comment (Masc999) [inline src/asmcnc/skavaUI/img/extraction_off.png:1]: Do we need this anymore? I can see it be done via overlay

File: src/asmcnc/comms/router_machine.py (hunk 2 above)
Comment (archiepeterjarvis) [reply src/asmcnc/comms/router_machine.py:1871]: hahaha yes I was rather confused

File: src/asmcnc/comms/router_machine.py (hunk 1 above)
Comment (archiepeterjarvis) [reply src/asmcnc/comms/router_machine.py:1188]: I think it's a very distinct thing from turning on the spindle, it's used for reading data. I see the issue with the naming, but I'd rather change the name than remove the function. But you have the last say ;)

**Review by Masc999 — APPROVED** [review]

**Pull request merged**